cmd/proxygenerator/interceptor.go

```diff
@@ -193,15 +215,15 @@ func visitPayloads(
 				if o == nil { continue }
 				no, err := visitPayload(ctx, options, parent, o)
 				if err != nil { return err }
-                objs[i] = no
```

Comment:
This file has some whitespace changes (this was using spaces which was looking bad even though the final generated source was always go fmt'd anyways)